jabgui/src/main/resources/org/jabref/gui/ai/components/aichat/AiChatComponent.fxml

```diff
@@ -44,6 +46,18 @@
                 <Tooltip text="%Notifications"/>
             </tooltip>
         </Button>
+        <MenuButton styleClass="icon-button,narrow">
+            <graphic>
+                <JabRefIconView glyph="SAVE"/>
+            </graphic>
+            <items>
+                <MenuItem text="%Export in human-readable format (Markdown)" onAction="#exportMarkdown"/>
+                <MenuItem text="%Export in machine-readable format (JSON)" onAction="#exportJson"/>
```

Comment:
```suggestion
                <MenuItem text="%Export to JSON" onAction="#exportJson"/>
```

Reason: JSON is also considered human readable - in comparison to a binary format